build.sbt

```diff
@@ -5,11 +5,11 @@ organization := "com.sclasen"
 name := "akka-zk-cluster-seed"
 version := "0.1.10-SNAPSHOT"
 
```

Comment:
@sclasen is this `version` value needs to be updated?

Reply:
not right now, I can publish a snapshot with these changes, and then if they work well, i'll bump to non snapshot and publish again